lib/Hakyll/Web/Pandoc.hs

```diff
@@ -68,6 +68,7 @@ readPandocWith ropt item =
         MediaWiki          -> readMediaWiki ro
         OrgMode            -> readOrg ro
         Rst                -> readRST ro
+        AsciiDoc           -> readAsciiDoc ro
```

Comment:
I think this is also going to require a little bit of CPP! Otherwise it won't compile for older Pandoc versions